app-network/fbthrift/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME=fbthrift
+PKGSEC=net
+PKGDEP="fizz fmt folly gflags glog mvfst wangle xxhash zlib zstd"
+PKGDES="RPC and serialization framework plus a code generator."
```

Comment:
Wh... what? I'm so confused.

Either way, at least drop the period point.

Reply:
From the upstream's README: Serialization and RPC framework for service communication